src/platform/linux.c

```diff
@@ -49,12 +53,21 @@ typedef enum { DT_NONE, DT_KDIALOG, DT_ZENITY } dialog_type;
 void platform_get_exe_path(utf8 *outPath, size_t outSize)
 {
 	char exePath[MAX_PATH];
+#ifdef __LINUX__
 	ssize_t bytesRead;
 	bytesRead = readlink("/proc/self/exe", exePath, MAX_PATH);
 	if (bytesRead == -1) {
 		log_fatal("failed to read /proc/self/exe");
 	}
 	exePath[bytesRead - 1] = '\0';
+#else
```

Comment:
Yes, you are correct, I have not noticed. I think I would still like you to include this guard though, of somebody in the future wants to put it to another platform, it would be easier to spot. Better yet: you could make this `elif _FreeBSD_` and still provide`else` clause with just `#error` in it.